deps/cJsonDemo/src/cJSONDEMO.c

```diff
@@ -40,7 +40,11 @@
 #pragma GCC visibility pop
 #endif
 
-#include "cJSONDEMO.h"
+#ifdef USE_CJSON
```

Comment:
1. suggest refactoring cJSONDEMO.c to make it work with V2 and V3 instead of using complicated cmake.
2. cJSONDEMO is not good as taosdemo is renamed to taosBenchmark. suggest a general name like tools_cJSON etc.